utils/deploy/envoy.yaml

```diff
@@ -337,7 +337,7 @@ data:
                                           "data": [
                                             {
                                               "selector": {
-                                                "selector": "filter_state.wasm\\.kuadrant\\.identity\\.userid",
+                                                "selector": "auth.identity.userid",
```

Comment:
This is probably not something one would ever want to do. 
All `selector` usages should be avoided. This should be an `expression` here.